pypiprivate/publish.py

```diff
@@ -5,14 +5,12 @@
 from pkg_resources import packaging
 from jinja2 import Environment
 
-
```

Comment:
Can you please discard these whitespace changes. Other than that the PR looks good to me